IMC_DataScience/IMC_OOPTutorial/blog.md

```diff
@@ -80,21 +91,23 @@ You’ll notice I added a getName() method that just returns the whole name, sin
 
 
 
-
 Using the method of an object is as simple as [specific object instance].[method name]. Here’s what the above code outputs:
 
 ![img](https://lh3.googleusercontent.com/-pn_m1XcMA0xH0wxxLkaechvajPV4lo0135r7WL1nuYdy1JOlXqmjEnoLveH8elh9gFi0HOKx0prMkTjZTcPt5bub14PbseULPN8DvhNivh38t-AATjeVxHnQTsnwge_NFpvMebE)
 
-Our last task for this is to change the age of the person. This one will use a method too, but it raises an important question: How exactly do we want to approach this? Do we want to give free access to increment the age, or do we want to simply increment it by one? An easy answer is to do both! 
-Quick Sublesson: Overloading
+
+
+Our last task for this is to modify the age of the person. This one will use methods too, but it raises an important question: How exactly do we want to approach this? Do we want to give free access to increment the age, or do we want to simply increment it by one? An easy answer is to do both! 
+
+###### 	Quick Sublesson: Overloading
 
 ​			*Note: This does not apply to every language! I know it works for C++ and Java, for example, but not Python. Do a quick google search to see if this will 			work for you. If not, skip this.*
 
-In some languages, two methods can be given the same name, if they take different inputs - as seen in the code example you’ll see in a second. This is called **overloading**.
+​	In some languages, two methods can be given the same name, if they take different inputs - as seen in the code example you’ll see in a second. This is 	called **overloading**.
 
-I’ve gone ahead and made a function that takes an input and adjusts the age accordingly, and another that takes no input and simply increments the age by one. With that, we have our complete (simple) Person.java file!
+I’ve gone ahead and made a function that sets the age to a new value, takes an input and adjusts the age accordingly, and another that takes no input and simply increments the age by one. With that, we have our complete (simple) Person.java file!
 
-![img](https://lh4.googleusercontent.com/NsFmGAZKJXQ0hWunbwB2_NVgK5-HcEt_rsFhNqD2tWJar0fbsUOXPpijWXj6dIlrlrX9BWGo87pm3OIsgF7W85zrfiE2J3b-YrugwbP1bQinJkZ3pFJcNiGRebTp0JZ8VUl0796s)
+![image-20200726115626085](C:\Users\MMO\AppData\Roaming\Typora\typora-user-images\image-20200726115626085.png)
 
 Our complete test program:
 
```

Comment:
Note: on line 120, you mention how Java has single inheritance, but this is not true for all languages. Java gets around this by being able to IMPLEMENT multiple interfaces.